spec/models/spree/tax_rate_spec.rb

```diff
@@ -386,6 +386,16 @@ module Spree
                 Spree::TaxRate.adjust(order, order.line_items)
                 expect(line_item.adjustments.credit.count).to eq 2
               end
+
+              it "notifies bugsnag" do
+                # there are two tax rate
+                expect(Bugsnag).to receive(:notify).with(
+                  "Notice: Tax refund should not be possible, please check the default zone and " \
+                  "the tax rate zone configuration"
+                ).twice
```

Comment:
I recently started adding `and_call_original` to Bugsnag expectations so that the code is actually run and errors in the metadata block are raised. That's only a theory though. I don't know if it would fail properly. But if my comment above is correct, you have the perfect case to test it.

Reply:
Unfortunately `and_call_original` did not pick up the error. I wouldn't be surprised if they rescue errors in the `notify` block so they can still report the error even if something is wrong.